connexion/operations/openapi.py

```diff
@@ -273,6 +273,9 @@ def _get_body_argument(self, body, arguments, has_kwargs, sanitize):
 
         if x_body_name in arguments or has_kwargs:
             return {x_body_name: res}
+        # If the content type is "multipart/form-data" return the properties as-is
+        if self.consumes[0] == 'multipart/form-data':
+            return res
         return {}
```

Comment:
I am pretty sure it might be sufficient to just return `res` in here. 
In method that calls the one you modified, `request.form.items` is put into `request_body`.

Reply:
Also, that's is needed for #987 to work.

Reply:
I just did it like this because I wasn't entirely sure if a non-empty response was always going to be the correct response. I can most certainly change it though.

Reply:
I pulled that version into #987 and there are some tests failing, so yours might be better TBH.
Rebase on top of master BTW, there are some changes that should make it better with tests and for you to see if direction is fine.